Engines/Wine/Verbs/d3dx9/script.js

```diff
@@ -1,6 +1,6 @@
 const Wine = include("engines.wine.engine.object");
 const Resource = include("utils.functions.net.resource");
-const {CabExtract} = include("utils.functions.filesystem.extract");
+const { CabExtract } = include("utils.functions.filesystem.extract");
```

Comment:
Why isn't this handled by the formatter?

Reply:
I have the habit of pressing the formatting keyboard shortcut directly before pressing the save shortcut. I think that this change comes from that.